tools/permission_tokens/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use std::collections::HashMap;
+
+use iroha_permissions_validators::public_blockchain::DefaultPermissionToken;
+use iroha_schema::{IntoSchema, Metadata};
+
+#[allow(clippy::print_stdout, clippy::expect_used, clippy::panic)]
+fn main() {
```

Comment:
Since this program is only used to generate permission tokens, maybe it should be added as a subcommand to `kagami`?